core/math/quaternion.cpp

```diff
@@ -83,7 +83,7 @@ Quaternion Quaternion::normalized() const {
 }
 
 bool Quaternion::is_normalized() const {
-	return Math::is_equal_approx(length_squared(), 1, (real_t)UNIT_EPSILON); //use less epsilon
+	return Math::is_equal_approx(length(), 1, (real_t)UNIT_EPSILON); //use less epsilon
```

Comment:
```suggestion
	return Math::is_equal_approx(length_squared(), 1, (real_t)UNIT_EPSILON); //use less epsilon
```
Since this issue depends on the accuracy of the inspector, the correct fix is to do it in the inspector, as in #106352.

If the accuracy of Quaternion's input in the inspector is increased, this line change is not necessary.